components/layouts/PageFooter/FooterSection.vue

```diff
@@ -1,41 +1,71 @@
 <template>
-  <section class="footer-column">
-    <FooterSubsection
-      v-for="el in elements"
-      :key="el.title"
-      :title="el.title"
-      :elements="el.elements"
-    />
-  </section>
+  <div class="footer-section">
+    <h2 class="footer-section__title">
+      {{ title }}
+    </h2>
+    <section :class="showIconLinks && 'footer-section__icons-group'">
+      <AppLink
+        v-for="element in elements"
+        :key="element.label"
+        :class="showIconLinks ? 'footer-section__icon-link' : 'footer-section__link'"
+        v-bind="element"
+      >
+        <component :is="element.icon" v-if="showIconLinks" />
```

Comment:
From the name of the property, I'd say this means "show the icons of the links _in addition_ to the labels" as if the default behaviour would be not showing them. Can we change this to something like `icons-only` to make it explicit we only want the icons? Then, as the default, we could render both if the icon is available.